src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -695,7 +706,7 @@ protected virtual bool TryLoadFromCache(string url, JsonOperationContext context
                             {
                                 var name = databaseMissing.FirstOrDefault();
                                 if (name != null)
-                                    throw new DatabaseDoesNotExistException(name);
+                                    throw new DatabaseDoesNotExistException($"Database {name} does not exists");
```

Comment:
Use DatabaseDoesNotExistException.Throw(name) here.

Reply:
I don't think it will be best for us here, when we throw the exception, you only see the url and the name of the database only in the inner exception you can see that this is DatabaseDoesNotExistException so I changed it to be more clear for the users

Reply:
Not true. Check https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Client/Exceptions/Database/DatabaseDoesNotExistException.cs#L33

Reply:
I checked it out, and then message I am getting is not enough to understand it,
Please check it out, just create a store and try to load from nonexistent database then you will see, wait for the last message the AggregateException.
localhost -> test that is what I am getting

Reply:
You should get https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Client/Exceptions/Database/DatabaseDoesNotExistException.cs#L52 message when
```
DatabaseDoesNotExistException.Throw(name);
```
is used?

Reply:
Didn't notice the the .Throw in the first commit I looked on the code and just completed the pieces I will change it Thanks

Reply:
You know it is the same message but for future use it is importent

Reply:
I know, but it's better to have one place to generate the message.